docs/consensus/proof_of_time.md

```diff
@@ -62,9 +64,8 @@ Takes a `seed` and number of iterations `pot_iterations` and repeatedly evaluate
     3. Set `input = output`. The `key` stays the same.
 5. Output `checkpoints`
 
-<!-- 
-TODO: replace diagram
-![Screenshot 2023-06-01 at 13.48.06.png](Dilithium%20PoT%20Specification%204e17a6d5b03a4abea864ba2d0b97970e/Screenshot_2023-06-01_at_13.48.06.png) -->
+![PoT Evaluator time in iteration.png](/img/PoT_Evaluator_time_in_iter.png)
```

Comment:
This should be this one https://github.com/subspace/subnomicon/blob/main/static/img/Proof_of_Time-light.svg and the corresponding dark one